tests/cpp/tree/hist/test_histogram.cc

```diff
@@ -317,11 +316,6 @@ TEST(CPUHistogram, BuildHistDistColSplit) {
   RunWithInMemoryCommunicator(kWorkers, TestBuildHistogram, true, false, true, false);
 }
 
-TEST(CPUHistogram, BuildHistDistColSplitSecure) {
```

Comment:
Why is this removed?

Reply:
It will not work and the tests implemented in TestBuildHistogram are not ready for secure tests - its logic is for non-secure version and does not cover the secure interface, etc. 
We need a test implementation compatible with secure schemes and add this (and RowSplitSecure) back